src/components/screens/send/index.js

```diff
@@ -23,8 +23,8 @@ const Send = ({ history }) => {
         finalCallback={backToWallet}
         className={styles.wrapper}
       >
-        <Form initialValue={initialValue} />
         <Summary />
+        <Form initialValue={initialValue} />
```

Comment:
Should this not be before summary? I think it will affect the steps order.

Reply:
Thank you. I forgot to change it back.